syncto/views/collection.py

```diff
@@ -0,0 +1,100 @@
+from pyramid import httpexceptions
+from pyramid.security import NO_PERMISSION_REQUIRED, forget
+
+from cliquet import Service
+from cliquet.errors import http_error, ERRORS, raise_invalid
+from sync.client import SyncClient
+
+from syncto import AUTHORIZATION_HEADER, CLIENT_STATE_HEADER
+from syncto.utils import base64_to_uuid4, uuid4_to_base64
+
+
+collection = Service(name='collection',
+                     description='Get the Firefox Sync Collection',
```

Comment:
This will do more than just "get" in the future, so we probably shouldn't use "get" in the description.